frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -36,6 +36,8 @@ import { featureFlagsSidebarLogic } from './sidebars/featureFlags'
 import { insightsSidebarLogic } from './sidebars/insights'
 import { personsAndGroupsSidebarLogic } from './sidebars/personsAndGroups'
 import { toolbarSidebarLogic } from './sidebars/toolbar'
+import { IconWeb } from 'lib/lemon-ui/icons'
```

Comment:
Not sure if this is something I should avoid? There isn't an obvious alternative here https://github.com/PostHog/icons

Reply:
Makes sense, have pinged, but feel free to close this PR if it gets handled elsewhere :)

Reply:
Have changed this to IconPieChart temporarily